src/extension/importers/roundFromMapsIplabs.ts

```diff
@@ -0,0 +1,125 @@
+import { RoundGame, RoundStore } from "types/schemas";
+import JSONCrush from "../../../node_modules/jsoncrush/JSONCrush.js" //blaming webpack for this
```

Comment:
If it imported this normally, webpack would bundle it using `require()`. That would cause issues upon starting nodecg, this does this happen if you import using the path directly.